Fix build with llvm 19:

rendering/software/function/array.h:432:59: error: a template argument list is expected after a name prefixed by the template keyword [-Wmissing-template-arg-list-after-template-kw]
  432 |                 { for(Iterator i(*this); i; ++i) i.get_array().template fill(x); }
      |                                                                         ^

Index: src/synfig/rendering/software/function/array.h
--- src/synfig/rendering/software/function/array.h.orig
+++ src/synfig/rendering/software/function/array.h
@@ -429,7 +429,7 @@ class Array: public Array<T, rank-1> { (public)
 	};
 
 	void fill(const Type &x) const
-		{ for(Iterator i(*this); i; ++i) i.get_array().template fill(x); }
+		{ for(Iterator i(*this); i; ++i) i.get_array().template fill<T>(x); }
 
 	template<typename TT>
 	void assign(const Array<TT, Rank> &x) const
